src/libraries/Microsoft.Win32.SystemEvents/src/CompatibilitySuppressions.xml

```diff
@@ -7,4 +7,16 @@
     <Left>lib/netstandard2.0/Microsoft.Win32.SystemEvents.dll</Left>
     <Right>lib/net462/Microsoft.Win32.SystemEvents.dll</Right>
   </Suppression>
-</Suppressions>
\ No newline at end of file
+  <Suppression>
+    <DiagnosticId>CP0016</DiagnosticId>
+    <Target>E:Microsoft.Win32.SystemEvents.EventsThreadShutdown:[T:System.ObsoleteAttribute]</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0016</DiagnosticId>
+    <Target>E:Microsoft.Win32.SystemEvents.EventsThreadShutdown:[T:System.ObsoleteAttribute]</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0016</DiagnosticId>
+    <Target>E:Microsoft.Win32.SystemEvents.EventsThreadShutdown:[T:System.ObsoleteAttribute]</Target>
+  </Suppression>
```

Comment:
These suppressions don't look right. I think you can remove them after you added the ObsoleteAttribute to the reference source.

Reply:
Tried this and I still received build errors:
```
C:\Users\lonitra\.nuget\packages\microsoft.dotnet.apicompat.task\10.0.100-alpha.1.24472.4\build\Microsoft.DotNet.ApiCom
pat.ValidateAssemblies.Common.targets(16,5): error : API breaking changes found. If those are intentional, the APICompa
t suppression file can be updated by rebuilding with '/p:ApiCompatGenerateSuppressionFile=true' [C:\Users\lonitra\sourc
e\repos\runtime\src\libraries\Microsoft.Win32.SystemEvents\src\Microsoft.Win32.SystemEvents.csproj]
```